includes/WpGears/Gearman/Worker.php

```diff
@@ -77,6 +77,8 @@ public function work() {
 			$result = false;
 		}
 
+		do_action( 'wp_async_after_worker_works', $result, $this );
```

Comment:
yah I wasn't sure if `task` was just a reference to the actions being in the do_job method rather than part of the namespace prefix for actions.  But I agree consistency is good, pushed the name change.